docker-compose.yml

```diff
@@ -185,9 +185,9 @@ services:
         max-size: "100m"
         max-file: "3"
     volumes:
-      - ./config/connectors/postgresql-42.6.0.jar:/usr/share/logstash/postgresql-42.6.0.jar
-      - ./config/logstash/logstash.conf:/usr/share/logstash/logstash.config
-      - ./config/logstash/ingest_change_table.sql:/usr/share/logstash/ingest_change_table.sql
+      - ./logstash/config/postgresql-42.6.0.jar:/usr/share/logstash/postgresql-42.6.0.jar
```

Comment:
Placing this JAR file under the "connectors" directory was intentional to illustrate what the file is. If we want to simplify the directory structure I'd suggest changing the filename to better indicate the file's purpose.

Reply:
great suggestion - change made. also pending conversation with Clay about how we want to handle pulling this file.